
try
    1,
    2
catch
    3
catch (a)
    console.log(a);
finally
    4;